packages/fhir-converter/src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -181,6 +181,32 @@ var getDateTime = function (dateTimeString) {
   ).toJSON();
 };
 
+const allValuesInObjAreNullFlavor = (obj) => {
```

Comment:
queue based approach to checking whether the entirety of an object is filled with nullFlavor values. Used in IF hbs primitive. 

Worth a second pair of eyes for any glaring bugs since this function decreased our dangling resource count greatly but is a large structural change applied on every "if" call in hbs templates